src/ape/api/convert.py

```diff
@@ -25,4 +25,22 @@ def convert(self, value: Any) -> ConvertedType:
         Convert the given value to the type specified as the generic for this class.
         Implementations of this API must throw a :class:`~ape.exceptions.ConversionError`
         when the item fails to convert properly.
+
+        Usage example::
+
+            from ape import convert
+            from ape.types import AddressType
+
+            convert("1 gwei", int)
+            # 1000000000
+
+            convert("1 ETH", int)
```

Comment:
Does this work?

Reply:
In [2]: convert("1 ETH", int)
Out[2]: 1000000000000000000

Reply:
The address does need the `ape-ens` plugin installed though to work properly otherwise it will show an error and I'll make a note of that in the other PR